packages/react-chat/src/components/ReactChat.tsx

```diff
@@ -1,6 +1,7 @@
 import React, { useRef } from "react";
 
 import { NarrowProvider } from "../contexts/narrowProvider";
+import { ThemeContextProvider } from "../contexts/themeProvider";
```

Comment:
ThemeProvider from styled components is enough no need to create custom context provider see [docs](https://styled-components.com/docs/advanced)